JustSaying.AwsTools.IntegrationTests/WhenIAccessAnExistingQueueWithoutAnErrorQueue.cs

```diff
@@ -9,7 +9,7 @@ public class WhenIAccessAnExistingQueueWithoutAnErrorQueue : WhenCreatingQueuesB
     {
         protected override void When()
         {
-            SystemUnderTest.Create(new SqsBasicConfiguration(), attempt: 0);
+            SystemUnderTest.Create(new SqsBasicConfiguration() {ErrorQueueOptOut = true}, attempt: 0);

```

Comment:
Had to add this to fix the broken integration test ... can someone confirm my assumed usage of ErrorQueueOptOut is correct?